modules/runner/src/main/ts/runner/Runner.ts

```diff
@@ -42,7 +42,7 @@ export const Runner = (rootSuite: Suite, params: UrlParams, callbacks: Callbacks
     const sum = reporter.summary();
     if (params.retry > 0) {
       params.retry = 0;
-      actions.updateHistory(params.offset, params.retry, sum.skipped);
+      actions.updateHistory(params.offset, sum.failed, params.retry, sum.skipped);
```

Comment:
It looks like the interface for the `updateHistory` fn is the following. I am wondering if there is an existing mistake somewhere since the `skipped` and `retry` arguments don't look to be in the right order? I could be missing something obvious though.
```
readonly updateHistory: (offset: number, failed: number, skipped: number, retry?: number) => void;
```

Reply:
Good catch thanks! I only noticed the failure argument was missing so I just added it, but you're right this had a few things wrong.